### PR TITLE
Mention special case of SyncTo for synchronisation

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
WFM

File: docs/api/1.x/synchronisation.rst [hunk 1]
```diff
@@ -138,6 +138,11 @@ and then fetch older records in background.
 
 .. image:: ../../images/sync-both.svg
 
+.. warning::
```
Comment (almet) [inline docs/api/1.x/synchronisation.rst]: I'm not sure we should add mozilla's specific doc here (this should probably leave in syncto instead)

Comment (leplatrem) [inline docs/api/1.x/synchronisation.rst]: Yep, I think you're right :)

**Comment by almet** [comment]
Let's close this then!

**Pull request closed without merging**